.github/workflows/avm.platform.publish.tag.yml

```diff
@@ -0,0 +1,63 @@
+name: "avm.platform.publish-tag"
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "The git tag of the module to publish."
```

Comment:
Would it be helpful to add an example?

Reply:
Maybe, but where? In the title of the input? Or as a comment in the code? 
There is an example in the utility script, and I guess we should add this to the AVM docs too. There is in general a User story open to describe the publishing process where this could be a part of.

Reply:
In the description

Reply:
That will make it very long. But ok, your call